.github/workflows/ci.yaml

```diff
@@ -20,6 +20,7 @@ on:
       - 'gh-readonly-queue/**'
       - 'release-**'
       - 'lts-**'
+      - 'backport-**'
```

Comment:
@hamzaremmal actually, do we care about builds of branches other than main? Typically a branch has a corresponding PR and then running the CI for this PR should be enough, right?

Reply:
Depends. We do sometimes maintain feature branches (like we did `cc-experiment`)...